test/compose.js

```diff
@@ -65,4 +65,14 @@ describe('compose', function() {
     assert.deepEqual(g(1, 2, 3), [1, 2, 3]);
   });
 
+  it('returns a curried function', function() {
+    function f(a) { return a + '_END'; }
+    function g(b, c, d) { return [b, c, d].join('_'); }
+    var c = R.compose(f, g);
```

Comment:
It would be nice to update the identifiers in one or other example to emphasize the equivalence between `R.compose(f, g)` and `R.pipe(g, f)`.